### DISCUSSION TITLE
Some standard tool beens

### DISCUSSION THREAD

**Discussion by Le0Michine** [kennetek/gridfinity-rebuilt-openscad#34] [Ideas]
I made a few relatively standard specialized tool bins and was wondering if that's something would would make sense to add here. For things like sanding disks (3 and 6 inch), hobby screwdrivers, bin with cylindrical cutouts to fit some relatively generic tools such as knives, screwdrivers etc. A bin for drill bits, though that would be pretty hard to make generic or parametric enough.

The pictures are from fusion but I'm pretty sure I could port them if there is some interest, I was planning on writing a script for sandpaper box anyway as it turned out to be quite cool and I need a few different configurations

![image](https://user-images.githubusercontent.com/2707008/206947593-da5f1aa6-7632-4a66-a79c-a721fc1857f1.png)

![image](https://user-images.githubusercontent.com/2707008/206947638-2f29e902-c6ad-4340-a6db-be88cb7678e0.png)

**Comment by kennetek** [comment]
Personally I'm fine with adding more specialized bins for items, which is why I put so much effort into the "module" structure of the repository. It is fairly simple to make a new file called `gridfinity-rebuilt-whatever.scad` that just imports `gridfinity-rebuilt-bins.scad` and then does all the specialized stuff in the new file. Basically, if its something that can be generalized (sanding disc diameter is a good one since its relatively simple, but even a screwdriver can have like 10 parameters that need to be measured by the user) then I think there's a use for it. However, things that are one-off models, like that fit a specific tool or item, should probably be kept to Printables/Thangs as a standalone STL, since the whole point of the scripts are to provide a easily customizable model. It all depends on the implementation I guess.
